src/workloads/scale/LLTInsert.yml

```diff
@@ -86,11 +86,11 @@ Actors:
           OperationName: RunCommand
           OperationCommand:
             setParameter: 1
-            transactionLifetimeLimitSeconds: 3600
+            transactionLifetimeLimitSeconds: 10800
 
 - Name: InitialLoad
   Type: Loader
-  ClientName: Loader
+  ClientName: Default
```

Comment:
Default sounds like a boring name. Is there a reason to it?

Reply:
no, I'll remove them